src/util/toast.ts

```diff
@@ -0,0 +1,11 @@
+import { toast as toastifyToast, ToastOptions } from 'react-toastify';
+
+/**
+ * Displays a toast which cannot be duplicated and
+ * continues ticking, even when the window loses focus.
+ */
+export const toast = (content: string, options: ToastOptions = {}) => {
+  if (!toastifyToast.isActive(content)) {
+    return toastifyToast(content, { ...options, ...{ toastId: content } });
```

Comment:
Small nit: I don‘t think you need the second spread here. You can directly assign the member. 

Also, I think I‘d prefer to allow overwriting the toastId with the parameters, so the spread order would need to change.

Reply:
Adapted!